packages/contracts/contracts/PCV.sol

```diff
@@ -179,6 +175,8 @@ contract PCV is IPCV, Ownable, CheckContract, SendCollateral {
         require(_council != council || _treasury != treasury, "PCV: these roles already set");
 
         changingRolesInitiated = block.timestamp;
+        governanceTimeDelay = thusdToken.getGovernanceTimeDelay();
```

Comment:
I'd suggest to have this as independent delay, same as in token contract

Reply:
Addressed in [refactor: add governance time delay in pcv constructor](https://github.com/Threshold-USD/dev/pull/113/commits/273a76adde2a0a3de6774338fedec2b873578bd4)